### DISCUSSION TITLE
OhMyPosh + PSReadLine's Vi?

### DISCUSSION THREAD

**Discussion by nlguerra2003** [JanDeDobbeleer/oh-my-posh#2366] [Troubleshoot] [answered]
I'm trying to see how to change OhMyPosh's behaviour depending on PSReadLine's Vi status.
For example, changing colours depending on the mode we're in.
Is there any way to do such things? I couldn't find anything related on the docs.

**Comment by JanDeDobbeleer** [comment, marked as answer]
@nlguerra2003 you want to have a look at [this](https://github.com/JanDeDobbeleer/oh-my-posh/issues/335#issuecomment-1032017862) one.
